### ISSUE TITLE
Module Template Path Not Dash Delimited

### ISSUE THREAD

**Issue by jbh** [zendframework/zend-expressive-tooling#82]
When I create a module using the Expressive CLI Tooling, the template path name in `ConfigProvider.php` is not dash delimited.

### Code to reproduce the issue

`composer expressive module:create DbExamples`

### Expected results

```
<?php
// src/DbExamples/src/ConfigProvider.php
// ...

/**
 * Returns the templates configuration
 */
public function getTemplates() : array
{
    return [
        'paths' => [
            'db-examples'    => [__DIR__ . '/../templates/'],
        ],
    ];
}

// ...
```

### Actual results

```
<?php
// src/DbExamples/src/ConfigProvider.php
// ...

/**
 * Returns the templates configuration
 */
public function getTemplates() : array
{
    return [
        'paths' => [
            'dbexamples'    => [__DIR__ . '/../templates/'],
        ],
    ];
}

// ...
```

**Comment by michalbundyra** [comment]
Hm... I am not sure about it. Is it causing any issues?

When we run command:

```bash
composer expressive module:create Db\\Examples
```

we will have `db-examples` as key in `paths`, but this is creating different module, on different level.

**Comment by jbh** [comment]
But I don't want `Db\\Examples`. I want `DbExamples`. It is causing issues, because the twig template engine looks for `db-examples` by default.

**Comment by jbh** [comment]
Test case:

```
composer create-project zendframework/zend-expressive-skeleton test-app
cd test-app
composer require --dev zendframework/zend-expressive-tooling
composer expressive module:create ExampleModule
composer expressive handler:create ExampleModule\\Handler\\IndexHandler
```

Take note about the different naming conventions:

```php
<?php
// ExampleModule\ConfigProvider
class ConfigProvider
{
    // ...
     * Returns the templates configuration
     */
    public function getTemplates() : array
    {
        return [
            'paths' => [
                'examplemodule'    => [__DIR__ . '/../templates/'],
            ],
        ];
    }
```

```php
<?php
// ExampleModule\Handler\IndexHandler
class IndexHandler implements RequestHandlerInterface
{
    // ...
    public function handle(ServerRequestInterface $request) : ResponseInterface
    {
        // Do some work...
        // Render and return a response:
        return new HtmlResponse($this->renderer->render(
            'example-module::index',
            [] // parameters to pass to template
        ));
    }
}
```

`ConfigProvider` defines a name without dashes, while the `Handler` is expecting a name with dashes.

I'm working on a solution now, but I wanted to put up a test case.

**Comment by jbh** [comment]
Interesting. `$expectedContent = sprintf($command::TEMPLATE_CONFIG_PROVIDER, 'MyApp', 'myapp');`

The test explicitly asserts that it should be sans dashes. Is this behavior on purpose? @weierophinney do you know any reason the template path name should be sans dashes?

**Comment by jbh** [comment]
I wonder if the handler creation should be sans dashes instead of adding dashes to the template namespace. Hm.

**Comment by weierophinney** [comment]
@jbh I think this is an area where the test was incorrect. The expectation should be a TitleCase to dash-separated conversion in all cases.

**Comment by jbh** [comment]
Thanks. I will make this change.

**Comment by michalbundyra** [comment]
Resolved in #83, released with 1.1.0.
